examples/bidirectional/shared/grpc.go

```diff
@@ -55,6 +55,7 @@ func (m *GRPCClient) Get(key string) (int64, error) {
 
 // Here is the gRPC server that GRPCClient talks to.
 type GRPCServer struct {
+	proto.UnimplementedCounterServer
```

Comment:
As an alternative to this, you can make this the default behaviour at generation time using the `require_unimplemented_servers=false` option.

Reply:
Oh thanks, TIL. I've switched to that now, as tbh I'd prefer to get build errors if I've forgotten to implement a method 🙂